src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InnvilgeVedtak.tsx

```diff
@@ -61,17 +46,10 @@ const Beregningstabell = styled(Utregningstabell)`
 `;
 export const InnvilgeVedtak: React.FC<{
```

Comment:
```suggestion
export const Vedtaksform: React.FC<{
```
Da blir det likt som i barnetilsyn

Reply:
https://github.com/navikt/familie-ef-sak-frontend/pull/2219/commits/a6bb9a63501a0129daa5d9b42b6be6c487d2bd3d